components/cli/cli/cli.go

```diff
@@ -50,6 +51,7 @@ type Cli interface {
 	CredManager() credentials.CredManager
 	CredReader() credentials.CredReader
 	Runtime() cliRuntime.Runtime
+	Sleep(seconds time.Duration)
```

Comment:
Why do we need Sleep as an interface function? Can't we directly use time.Sleep?

Reply:
Previously we directly used time.Sleep. However now when this function is getting called from unit tests, they are being kept waiting for that same period of time unnecessarily.